client/starter-app/src/app/app-routing.module.ts

```diff
@@ -1,15 +1,35 @@
 import { NgModule } from '@angular/core';
+import {ModuleWithProviders} from '@angular/core'
```

Comment:
import the modules with a ',' since they are both from '@angular/core'
import { NgModule, ModuleWithProviders } from '@angular/core'